src/ts/content/entities/bookForm/render.ts

```diff
@@ -43,7 +43,7 @@ const handleFormMutation = () => {
 	}
 };
 
-window.addEventListener("load", () => {
+window.addEventListener("pageshow", () => {
```

Comment:
why the switch to pageshow vs load lol

Reply:
Firefox can't use page load because if the page is already loaded (i.e. cached) on navigation to the site, the load event won't be fired.